### PR TITLE
[zh-cn]: create the translation of `tabs.duplicate()` method

### PR DESCRIPTION
**Pull request by tianyeeT**

### CODE REVIEW

**Comment by tianyeeT** [comment]
Related PR: https://github.com/mdn/content/pull/34312
I changed the original `Optional` into `{{optional_inline}}`.

File: files/zh-cn/mozilla/add-ons/webextensions/api/tabs/duplicate/index.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+---
+title: tabs.discard()
+slug: Mozilla/Add-ons/WebExtensions/API/tabs/discard
```
Comment (tianyeeT) [inline files/zh-cn/mozilla/add-ons/webextensions/api/tabs/duplicate/index.md:3]:
```suggestion
title: tabs.duplicate()
slug: Mozilla/Add-ons/WebExtensions/API/tabs/duplicate
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/tabs/duplicate/index.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  - : `object`。一个描述如何复制标签页的对象。它包含以下属性：
+
+    - `index` {{optional_inline}}
+      - : `integer`。新标签页在窗口中的位置。该值的范围限制为零到窗口中标签页的数量。
+    - `active` {{optional_inline}}
+      - : `boolean`。标签页是否成为窗口中的活动标签页。不影响窗口是否被聚焦。默认为 `true`。
+
+### 返回值
+
+一个[`Promise`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise)，其会在复制标签页后兑现一个 {{WebExtAPIRef('tabs.Tab')}} 对象，描述复制的标签页的详细信息。如果扩展具有 [`"tabs"` 权限](/zh-CN/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions)或匹配的[主机权限](/zh-CN/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions#host_permissions)，`Tab` 对象只包含 `url`、`title` 和 `favIconUrl`。如果发生任何错误，则该 promise 将被拒绝并带有错误消息。
```
Comment (yin1999) [inline files/zh-cn/mozilla/add-ons/webextensions/api/tabs/duplicate/index.md:38]:
```suggestion
一个 [`Promise`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise)，其会在复制标签页后兑现一个 {{WebExtAPIRef('tabs.Tab')}} 对象，描述复制的标签页的详细信息。如果扩展具有 [`"tabs"` 权限](/zh-CN/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions)或匹配的[主机权限](/zh-CN/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions#主机权限)，`Tab` 对象只包含 `url`、`title` 和 `favIconUrl`。如果发生任何错误，则该 promise 将以错误消息拒绝。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/tabs/duplicate/index.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+复制第一个标签页，然后记录新创建的标签页的 ID：
+
+```js
+function onDuplicated(tabInfo) {
+  console.log(tabInfo.id);
+}
+
+function onError(error) {
+  console.log(`发送错误：${error}`);
```
Comment (yin1999) [inline files/zh-cn/mozilla/add-ons/webextensions/api/tabs/duplicate/index.md:52]:
```suggestion
  console.log(`发生错误：${error}`);
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

**Review by yin1999 — APPROVED** [review]

**Pull request merged**